rollbar-java/build.gradle

```diff
@@ -16,6 +16,8 @@ dependencies {
 
     compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.25'
 
+    compile 'com.google.code.findbugs:jsr305:3.0.2'
```

Comment:
Why is this dependency added?

Reply:
I think we change that dependency to `compileOnly`? You will need to include it from your project if Kotlin needs it at runtime.